src/classes/RD2_ElevateInformation_CTRL.cls

```diff
@@ -59,19 +56,45 @@ public with sharing class RD2_ElevateInformation_CTRL {
     public static Map<String, Object> getData(Id recordId) {
         Map<String, Object> data = new Map<String, Object>();
 
+
```

Comment:
remove cache = false

Reply:
Why remove cache=false? The annotation parameter should always be included, but specified as either true or false. Plus, if this is cachable does that impact the retrieval of the error message? If the user refreshes the page this [cached] method may not be re-retrieved which would mean a new error wouldn't render. That's why I was wondering if the error needed to be separate from the permissions data.

Reply:
I remove the cache=false because by default the cache will be set to false. I agree with the separating error message and permission data to 2 different methods and only have error retrieval to be non-cacheable.